docs/en/tutorials/embeddables.rst

```diff
@@ -77,6 +77,63 @@ the embedded object.
         $this->address = new Address();
     }
 
+Nullable Embeddables Hydration
+------------------------------
+In case all fields in the embeddable are null, the entity is hydrated with an empty
+embeddable object. Properties in nullable embeddables should have
+a default value. You can normalize entities on ``postLoad`` event
+(in Doctrine ORM lifecycle events), to avoid getting the empty embeddable object
+instead of a null value.
+
+.. configuration-block::
+
+    .. code-block:: attribute
+
+        <?php
+        use Doctrine\ORM\Mapping\Column;
+        use Doctrine\ORM\Mapping\Entity;
+        use Doctrine\ORM\Mapping\Embeddable;
+        use Doctrine\ORM\Mapping\Embedded;
```

Comment:
Let's use the lexicographic order please.

```suggestion
        use Doctrine\ORM\Mapping\Column;
        use Doctrine\ORM\Mapping\Embeddable;
        use Doctrine\ORM\Mapping\Embedded;
        use Doctrine\ORM\Mapping\Entity;
```

Reply:
Thank you for your tips.